### DISCUSSION TITLE
What should a CSV file contain?

### DISCUSSION THREAD

**Discussion by Adam-Albu** [LemonFoxmere/DeepFusion#10] [Q&A]
What should a CSV file contain? I've got one already and it looks like this:
input,target
1,2
2,3
3,4
4,5
5,6

But it's giving me a parse error. Am I doing something wrong or is a system error?

**Comment by LemonFoxmere** [comment]
Can you provide some more details about the parse error? And also just on a side note: your input and output data should be on separate CSV files. The input CSV should be uploaded to the input node, and the output CSV to the output node.
